DevUtils/parse-langtags/langtags.schema.json

```
@@ -0,0 +1,177 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$ref": "#/definitions/langtags",
+  "definitions": {
```

```suggestion
  "$comment": "See https://gist.github.com/mcdurdin/eff65bb2d77034525120d48cbe280d30"
  "definitions": {
```

Let's add a comment to the schema referring back to where it came from

---

I'd really like to see the schema defn adopted by the langtags repo @mhosken 😁

---

See https://github.com/silnrsi/langtags/blob/master/source/langtags_schema.json now for official schema source